roles/uc-ui/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+uc_ui_listen_address: 0.0.0.0
+uc_ui_port: 9296
```

Comment:
Do you have any use case for this?

Reply:
No it was just mimicking the other services so removing it for now.